api/swagger.yaml

```diff
@@ -7272,20 +7274,6 @@ paths:
               description: |
                 A base64 - encoded JSON object with some filesystem header
                 information about the path
-        400:
-          description: "Bad parameter"
-          schema:
-            allOf:
-              - $ref: "#/definitions/ErrorResponse"
```

Comment:
Oh! I should've been clearer; the `HEAD` and `GET` endpoints can still return a `400` status, but it will be a "regular" one (so a plain `#/definitions/ErrorResponse`, same as other endpoints); this may happen on this line (for example); https://github.com/moby/moby/blob/7b9275c0da707b030e62c96b679a976f31f929d3/api/server/router/container/copy.go#L78-L79